__tests__/api.test.js

```diff
@@ -229,3 +228,116 @@ describe("POST to /api/articles/:article_id/comments", () => {
+  });
+  it("should return with an updated article for a number greater that 1", async () => {
+    const input = { inc_votes: 10 };
+    const articleBefore = {
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: "2020-07-09T20:11:00.000Z",
+      votes: 100,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    }; // this is the article that we will be checking in this test
+    const { body } = await request(app)
+      .patch("/api/articles/1")
+      .send(input)
+      .expect(200);
+    expect(body).not.toEqual(articleBefore);
+    expect(body).toEqual({
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: "2020-07-09T20:11:00.000Z",
+      votes: 110,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    });
+  });
+  it("should return an error if the article id doesnt exist", async () => {
+    const input = { inc_votes: 10 };
+    const { body } = await request(app)
+      .patch("/api/articles/29020202")
+      .send(input)
+      .expect(404);
+    expect(body.message).toBe("Article ID does not exist");
+  });
+  it("should return an error if the input object has the wrong properties", async () => {
+    const input = { inc_votes2: 2 };
+    const { body } = await request(app)
+      .patch("/api/articles/29020202")
+      .send(input)
+      .expect(400);
+    expect(body.message).toBe("Bad Request");
+  });
+  it("should return an error if the input is of the wrong format", async () => {
+    const input = { inc_votes2: "hello" };
+    const { body } = await request(app)
+      .patch("/api/articles/2")
+      .send(input)
+      .expect(400);
+    expect(body.message).toBe("Bad Request");
+  });
+});
+
+describe("DELETE to /api/comments/:comment_id", () => {
+  it("should delete a given comment by the comment id, respond with status 204 and no content", async () => {
+    const { body } = await request(app).delete("/api/comments/1").expect(204);
+    expect(body).toEqual({});
+  });
+  it("should return an error if the comment id is invalid", async () => {
+    const { body } = await request(app)
+      .delete("/api/comments/1121312313")
+      .expect(404);
+    expect(body.message).toBe("comment does not exist");
+  });
+  it("should return an error if the comment id is of the wrong type", async () => {
+    const { body } = await request(app)
+      .delete("/api/comments/hello")
+      .expect(400);
+    expect(body.message).toBe("Bad Request");
+  });
+  it("should delete that comment belonging to the id and it should no longer be in the database", async () => {
```

Comment:
We shouldn't need comments to make the test readable, without these comments does this test work? It may not be needed or could potentially be written a bit differently if you did want to include it.